src/include/dawg/details/xorgen4096_64.h

```diff
@@ -66,44 +60,44 @@ struct xorgen4096_64_mutt_gen {
 		w = s.w;
 		i = s.i&63;
 	}
-	
-	inline void seed(boost::uint32_t xx) {
+
+	inline void seed(std::uint32_t xx) {
 		i = 0;
-		w = UINT64_C(0x61C8864680B583EB);
-		native_t x = UINT64_C(0x6A7BCC427F295846);
+		w = static_cast<std::uint64_t>(0x61C8864680B583EB);
```

Comment:
This is incorrect. The UINT64_C is not the same as casting a value to a unit64_t.

However, with C++11, we can use the suffix `ull` to guarantee that it will be a 64-bit literal